src/components/Home.css

```diff
@@ -0,0 +1,18 @@
+.welcome-container {
+  width: 100%;
+  height: 80%;
+  padding: 1rem;
+  margin-top: 10vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  color: #fff;
+  border: 3px solid #f80;
+  box-shadow: 0 0 4px 4px rgba(197, 197, 197, 0.7);
+  background-color: rgba(0, 0, 0, 0.8);
+}
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Hello, @HAMMAS-SALEEM thank you so much, and I will keep in mind your recommendation